libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -172,6 +172,38 @@ def construct_analyzer_cmd(self, res_handler):
             LOG.error(ex)
             return []
 
+    def get_analyzer_mentioned_files(self, output):
```

Comment:
Shouldn't we depend on the files listed in the plist output instead of parsing the clang sa output?